languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -8316,18 +8316,21 @@ USA
 
         <rule id='TOMAR_ASSUMIR' name="[Universitário][Científico] V. Tomar → V. Assumir" tone_tags="academic" default="temp_off">
             <pattern>
+                <token postag='AQ.+|NC.+|NP.+|CS|CC' postag_regexp='yes'>
+                    <exception postag_regexp='no' postag='RG'/> <!-- Add more exceptions here as they are found -->
+                </token>
```

Comment:
It appears to me that the problem is the double meaning of “tomar”—the rule doesn't work when it is used literally (e.g., for drinks, medication, etc.). So the question is, does this exception fix this issue? Or what does it exclude?

We'd need quite a long list of exceptions in `<token postag='AQ.+|NC.+' postag_regexp='yes'/>` to avoid the literal false alarms. Some I can think of: `drinks|bebidas|cafés|su[mc]os|vinhos|cervejas|sorvetes|gelados|remédios|medicações|medicamentos|shots|chás|táxis|carros|ônibus|autocarros|trens|comboios|voos`, etc.